docs/user/BYOnetwork.md

```diff
@@ -0,0 +1,23 @@
+## Supported scenarios and requirements for using existing AWS network resources
+
+The table below shows the scenarios supported with using existing/bring your own(BYO) network resources:
+
+| <div style="width:70px">Scenario</div> | <div style="width:70px">BYO resource</div> | <div style="width:500px">Requirements</div> | <div style="width:100px">Resources that will be created</dvi>|
```

Comment:
Can we change the title of the second column from "BYO resource" to "Required BYO resource"

Reply:
These are optional choices and technically not required, so it's not confusing.

Reply:
I meant required in the context of the scenarios in this table only vs always required. I like the change you made in another commit adding required in a few other places;.

Reply:
I'm still on the fence about this

Reply:
See latest

Reply:
I like it thanks!